JSDemos/Demos/Charts/SankeyChart/description.md

```diff
@@ -1 +1,19 @@
-The DevExtreme JavaScript Sankey Chart is&nbsp;used to&nbsp;visualize the flow magnitude between value sets. The values are connected. A&nbsp;wider connection denotes a&nbsp;higher flow magnitude.
\ No newline at end of file
+The Sankey chart visualizes the flow magnitude between value sets. Flow values are nodes, the connections between nodes are links. The higher the flow magnitude, the wider the link is.
```

Comment:
What do you mean by "flow magnitude"?
Is it a widely-used technical term in a programming context? (I wasn't able to find instances of this phrase in the Microsoft documentation.)

Reply:
In the context of a Sankey diagram both termins "flow" and "magnitude" are widely used. The Sankey is a type of a flow diagram, where the lines/arrows/links (they are called differently in different sources) represent the magnitude of the flow. So I think it's okay to call it the "flow magnitude" instead of the "magnitude of the flow".

Reply:
Okay, thank you for the clarification. I'd like to add the word "and" before "the connections":

```suggestion
The Sankey chart visualizes the flow magnitude between value sets. Flow values are nodes, and the connections between nodes are links. The higher the flow magnitude, the wider the link is.
```